src/pages/jetton/context/JettonActionsContext.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { createContext, useState } from "react";
+
+const JettonActionsContext = createContext({
```

Comment:
I think recoil would be better for this. contexts are usually inefficient for rendering if they're value is an object that changes (and also we just don't really use them as a standard). They're good as far as I'm aware for static/nearly-static properties such as user's language, theme etc.

So up to you whether to refactor, but let's avoid them altogether moving forward.